src/components/BackButton/index.jsx

```diff
@@ -37,17 +51,13 @@ const BackButton = ({
       .slice(0, -1)
       .join('/')
   }
-  const handleClick = (onClick = onClick || (() => to && router.push(to)))
   const arrowColor =
     theme === 'primary'
       ? getCssVariableValue('primaryContrastTextColor')
       : getCssVariableValue('coolGrey')
+  const handleClick = (onClick = onClick || (() => to && router.push(to)))
   return isMobile ? (
-    <BarLeft>
-      <button className="coz-bar-btn coz-bar-burger" onClick={handleClick}>
-        <Icon icon={arrowLeft} color={arrowColor} />
-      </button>
-    </BarLeft>
+    <BackButtonMobile onClick={this.handleClick} theme={theme} />
```

Comment:
You don't used the const `handleClick` ?

Reply:
Fixed in a next PR, thx